tests/unit/StringsTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Tests\Unit;
+
+use PHPUnit\Framework\TestCase;
+use GeeksAreForLife\Utilities\Strings;
+
+class StringsTest extends TestCase
+{
+    /** @test */
+    public function string_starts_with_str(): void
```

Comment:
So, typically I'd name tests with the behaviour of the method, and if you need to run the same test with different data sets to verify it, look into data providers. A very useful blog post is here: https://tighten.co/blog/tidying-up-your-phpunit-tests-with-data-providers/

So I'd write a test like:

```
/**
 * @test
 * @dataProvider strings_start_with_values
 */
public function it_returns_true_if_strings_start_with_values($inputString, $startsWith) {
    $this->assertTrue(Strings::startsWith($inputString, $startsWith));
}
```

I'll let you look into the data providers in the linked blog post, but note the section on adding a descriptive identifier for the data being provided, as that can give descriptive names in test outputs.

You can then write similar (with a different data provider) for strings that **don't** start with other strings.

Reply:
That makes perfect sense, I think!  I have updated, is this what you mean?